### PR TITLE
Show terms and conditions based on feature flag

### PR DESCRIPTION
**Pull request by lserra-iov**

### CODE REVIEW

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/common/components/common/TermsDialog.vue [hunk 1]
```diff
@@ -11,21 +11,7 @@
...
-              </p>
-              <h3>
-              1914 translation by H. Rackham
-              </h3>
-              <p>
-                "But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was born and I will give you a complete account of the system, and expound the actual teachings of the great explorer of the truth, the master-builder of human happiness. No one rejects, dislikes, or avoids pleasure itself, because it is pleasure, but because those who do not know how to pursue pleasure rationally encounter consequences that are extremely painful. Nor again is there anyone who loves or pursues or desires to obtain pain of itself, because it is pain, but because occasionally circumstances occur in which toil and pain can procure him some great pleasure. To take a trivial example, which of us ever undertakes laborious physical exercise, except to obtain some advantage from it? But who has any right to find fault with a man who chooses to enjoy a pleasure that has no annoying consequences, or one who avoids a pain that produces no resultant pleasure?"
-              </p>
+              <p>{{ $props.text }}</p>
```
[the remark this answers is not in the document]
Comment (lserra-iov) [reply src/common/components/common/TermsDialog.vue:14]: New line characters  (`\n`, `\r `or `\r\n`) are interpreted as basic whitespaces. We could convert them into multiple p-tags with some additional processing.

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by ronaldsg20 — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Pull request merged**